packages/expo-ads-admob/README.md

```diff
@@ -34,7 +34,13 @@ In your app's `Info.plist` file, add a `GADApplicationIdentifier` key with a str
 
 ### Configure for Android
 
-No additional set up necessary.
+In your app's `AndroidManifest.xml` file, add a `meta-data` element with a string value of your AdMob app ID, as shown in [Google's Mobile Ads SDK (Android) documentation](https://developers.google.com/admob/android/quick-start#update_your_androidmanifestxml).
+
+```xml
+<meta-data
+            android:name="com.google.android.gms.ads.APPLICATION_ID"
+            android:value="ca-app-pub-xxxxxxxxxxxxxxxx~yyyyyyyyyy"/>
```

Comment:
```suggestion
<manifest>
  <application>
    ...
    <!-- Ensure that tag with this name and proper value is inside application -->
    <meta-data
      android:name="com.google.android.gms.ads.APPLICATION_ID"
      android:value="ca-app-pub-xxxxxxxxxxxxxxxx~yyyyyyyyyy"/> <!-- App ID -->
    <!-- You can find your App ID in the AdMob UI -->
    ...
  </application>
</manifest>
```

Reply:
Thanks for the suggested changes. I think it'd be good to keep the section consistent with the "Configure for iOS" section, which is what my original commit reflected. Not a big deal either way, but consider updating the "Configure for iOS" section to be consistent with your suggested changes.